src/main/java/org/gelecekbilimde/scienceplatform/common/util/ListUtil.java

```diff
@@ -0,0 +1,15 @@
+package org.gelecekbilimde.scienceplatform.common.util;
+
+import lombok.experimental.UtilityClass;
+
+import java.util.List;
+
+@UtilityClass
+public class ListUtil {
+
+	@SuppressWarnings({"unchecked", "This method is unused by the application directly but Spring is using it in the background."})
+	public static <C> List<C> to(Object object, Class<C> clazz) {
```

Comment:
Bu yazım yanlışı mı, yoksa bilerek mi böyle yazıldı ?

Reply:
Bilerek öyle yazıldı, çünkü java `class` yazmaya izin vermiyor. Bu yüzden genelde java yazanlar `clazz` ifadesini kullanırlar. 

https://stackoverflow.com/questions/2529974/why-do-java-programmers-like-to-name-a-variable-clazz 👀